server/src/test/java/org/candlepin/policy/AutobindRulesTest.java

```diff
@@ -2650,7 +2661,7 @@ public void testShouldSelectPoolWhenSLAMatchesButOtherSysPurposeAttributesAndVcp
 
         // Candidate pools:
         Product prodMCT1650 = createSysPurposeProduct(null, "random_role", "random_addon",
-            "mysla", "random_usage");
+            "mysla", "random_usage", null);
```

Comment:
Same here: This should be "random_service_type" instead of null, since the test expects all attributes other than SLA to have some random value

Reply:
Done.